assets/frpc/embed.go

```diff
@@ -0,0 +1,14 @@
+package frpc
```

Comment:
You'd better not move static files into this directory. Keep it in a independent static directory.